datacenter/dtr/2.2/guides/admin/monitor-and-troubleshoot/troubleshoot-with-logs.md

```diff
@@ -47,21 +47,25 @@ DTR replica to check the DTR internal state.
 {: .warning }
 
 Use SSH to log into a node that is running a DTR replica, and run the following
-command, replacing `$REPLICA_ID` with the ID of the DTR replica running on that
-node:
+commands:
 
 ```none
+# REPLICA_ID will be the replica ID for the current node.
+REPLICA_ID=$(docker volume ls -qf name=dtr-rethink |cut -d- -f3)
```

Comment:
I wouldn't use this to grab the `REPLICA_ID` variable, it's inconsistent, especially if the customer has situations where they once had a previous replica on that node and the underlying volume wasn't properly pruned.

The better way to grab it is to just look at the `Replica` value from `com.docker.compose.project:` and use `awk` or `sed` to cut that down.

Here's the line from an example inspect:
~~~
com.docker.compose.project: “Docker Trusted Registry 2.2.4 - (Replica 4d183c9a058f)”
~~~

And here's an example of grabbing the replica id from that line:
~~~
REPLICA_ID=$(docker inspect $(docker ps -aq --filter name=dtr-rethink) | grep 'com.docker.compose.service' | awk -F'rethinkdb-' '{print substr($2, 1, length($2)-2)}'
~~~

Reply:
This is shorter and does the same thing:

```
docker ps -lf name='^/dtr-rethinkdb-.{12}$' --format {{.Names}} |cut -d- -f3
```

Acceptable?